spec/features/blacklight_customizations/bookmarks_text_spec.rb

```diff
@@ -10,19 +10,20 @@
   end
 
   scenario "has bookmarks text as select" do
-    within first("div.documentHeader") do
+    within first("div.col-2") do
```

Comment:
Is this still useful? I think I'd take this out.

Reply:
I think it is. You can tell that the bookmarks are working properly. Or are you suggesting you remove the within statement?

Reply:
Yes, just the within statement. I don't think a feature test should care that there's a `div` most of the time.